adding tidyhydat blog post

boshek opened the pull request:
@sckott @stefaniebutland - for review

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+authors:
+- name: Sam Albers
+  url: https://github.com/bcgov
+date: 2018-01-23
```
date: 2018-01-16

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+
+One of the best things about learning R is that no matter your skill level, there is always someone who can benefit from your experience. Topics in R ranging from complicated machine learning approaches to calculating a mean all find their relevant audiences. This is particularly true when writing R packages. With an ever evolving R package development landscape (R, GitHub, external data, CRAN, c ...
+
+`tidyhydat` is a new rOpenSci R package that provides a standard method of accessing [Environment and Climate Change Canada's](https://www.canada.ca/en/environment-climate-change.html) [Water Survey of Canada](https://wateroffice.ec.gc.ca/index_e.html) data sources (WSC) using a consistent and easy to use interface that employs [tidy data principles](https://cran.r-project.org/web/packages/tidyr/ ...
```
you spell out "Water Survey of Canada" after defining abbreviation. decide which to do throughout post

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+
+# Acknowledgements
+
+I would like to thank rOpenSci for the opportunity to have the `tidyhydat` package reviewed. In particular I would like to thank [SCott Chamberlain](https://github.com/sckott), [Laura DeCicco](https://github.com/ldecicco-USGS) and [Luke Winslow](https://github.com/lawinslow) for their time and efforts in improving the package. Hopefully this post has outlined the ways in which their contributions ...
```
Scott (fix the C)

boshek replied:
done

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+![](../../themes/ropensci/static/img/blog-images/2018-01-23-tidyhydat/sp_plt-1.png)<!-- -->
+
+# What does the plot tell us
+Our original objective was to assess if the `08MH024` station was a **unique** station. This type of plot provides a clear pattern of hydrologic linkages. Flows at `08MH024` are more similar to stations at higher elevations (e.g. `08MH016`) than lower elevation tributaries (e.g. `08MH002`). This likely reflects the importance of upstream hydrologic process. The Fraser River, the largest river in  ...
```
(228000 km^2^) superscript doesn't render in the preview. Any ideas @sckott ?

stefaniebutland replied:
You don't say exactly whether or not it's unique

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+`tidyhydat` is a new rOpenSci R package that provides a standard method of accessing [Environment and Climate Change Canada's](https://www.canada.ca/en/environment-climate-change.html) [Water Survey of Canada](https://wateroffice.ec.gc.ca/index_e.html) data sources (WSC) using a consistent and easy to use interface that employs [tidy data principles](https://cran.r-project.org/web/packages/tidyr/ ...
+
+## Installation
+<img src="../../themes/ropensci/static/img/blog-images/2018-01-23-tidyhydat/tidyhydat_large.png" alt="Drawing" style="width: 200px;"/>
```
I think you can make path: /img/blog-images/2018-01-23-tidyhydat/tidyhydat_large.png. Same for other image

With change of post date, you'll need to change date of dir for images too

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+
+## 2. Where to store external data
+
+The single biggest feature of `tidyhydat` that caused the most headaches and learning opportunities for me is the fact that much of it's capabilities rely on an external `.sqlite3` data: HYDAT. HYDAT is the WSC's quarterly published [archive of hydrometric data](http://collaboration.cmc.ec.gc.ca/cmc/hydrometrics/www/). Some stations data range extends into the 19^th^ century. It is, however, rath ...
```
"its" no apostrophe

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
````diff
@@ -0,0 +1,215 @@
+```
+
+## 1. Documentation
+A second and third set of eyes, in the form of rOpenSci reviewers, on the tidyhydat documentation added to the quality of the package immensely. It is so easy to write documentation that makes sense when you are writing the code, and makes zero sense to anyone else. Having reviewers review documentation and politely tap you on the shoulder to say "this is barely English" is the clear line between ...
````
`tidyhydat` in backticks in this para

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+## 5. An exercise in reproducible science.
+`tidyhydat` was originally developed to save time accessing data. Instead of frequently opening database connections, making approximately the same query, forgetting to close the database connection, trying to reopen it, cursing R and generally blaming all the wrong people, I developed a package.  However, after using the package it has become clear that it has a much deeper and useful feature &m ...
+
+# A reproducible work flow
```
"how unique is this station to others" is missing word "compared"

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+To illustrate a reproducible workflow facilitated by `tidyhydat` and made possible by the capabilities of R, we can ask a simple question related to a hydrometric station: how unique is this station to others in the same watershed? In this example we will consider the FRASER RIVER AT MISSION station. Records extend over many years and we can compare an overlapping record using correlation to deve ...
+
+## Required packages
+A portion of the package audience is likely those with little or no experience in R so an example of the type of analysis possible with tidyhydat is helpful. For our purposes here, we need to install a whole other slew of packages. The packages needed for this analysis mostly fall into either the `tidyverse` collection of packages or the `igraph` family of packages. These packages can be installe ...
```
- put tidyhydat in backticks here.

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
````diff
@@ -0,0 +1,215 @@
+```r
+install.packages("tidyhydat")
+```
+
````
I'm wondering if here, you could have heading # Five things

stefaniebutland commented on content/blog/2018-01-23-tidyhydat.md:
```diff
@@ -0,0 +1,215 @@
+
+`tidyhydat` is a new rOpenSci R package that provides a standard method of accessing [Environment and Climate Change Canada's](https://www.canada.ca/en/environment-climate-change.html) [Water Survey of Canada](https://wateroffice.ec.gc.ca/index_e.html) data sources (WSC) using a consistent and easy to use interface that employs [tidy data principles](https://cran.r-project.org/web/packages/tidyr/ ...
+
+## Installation
```
Could you do without this heading and just say "tidyhydat can be installed ..."? The little code block will draw attention to this bit anyway without having a heading for one line of text

boshek commented:
I may have done this incorrectly. I made the changes and just pushed them up. I agree with all of @stefaniebutland and tried to address each comment.

stefaniebutland commented:
You did it right. Changes look good. Scott will review Monday (away rn)

Thanks for the nice words Sam 😊. Happy weekend

boshek commented:
@sckott Struggling to center the logo. Maybe it doesn't matter and looks fine the way it is. Also have not managed to the resolve the superscript issue. Otherwise I am done editing.

sckott commented:
having a look now

sckott commented:
@boshek 

- for centering logo, you can do `<center><img src="/img/blog-images/2018-01-16-tidyhydat/tidyhydat_large.png" alt="Drawing" style="width: 300px;" /></center>`
- let's move all headers to one smaller (so add another `#`) for everyone of them - they are quite large right now.
- for images, use `/img/blog-images/ etcetera` for them to show up correctly
- for superscripts i'm not sure how to make that work. can maybe just do `19th` century as a compromise

boshek commented:
@sckott 
* Headers/Centering -> yep
* Which images are not displaying correctly?
* I'm fine with 19th. Are we also ok with km2 instead of km<sup>2</sup>?

sckott commented:
Both images

Hmm, yeah I guess km2 doesn't make as much sense. I'm not sure how to fix it.

boshek commented:
@sckott 
* Fixed headers
* Fixed image links - displays fine on my machine (windows 7, chrome)
* Changed to square km. A bit hacky but since this is the only instance, I don't mind.

sckott commented:
think it's good to go now @stefaniebutland

stefaniebutland commented:
Thanks @sckott and @boshek. This will be posted this evening (= Jan 16th in some timezones) and will tweet from rOpenSci Tues AM 😄
